### PR TITLE
refactor: Prefer own implementation of app activation over idb one

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
I've checked how `focus` command is implemented in idb code and ported it to PyObjC

### CODE REVIEW

File: lib/utils.js [hunk 1]
```diff
@@ -289,6 +301,28 @@ async function getDeveloperRoot () {
...
+ * while activating the app
+ */
+async function activateApp (pid) {
+  const tmpScript = await tempDir.path({
+    prefix: `activate_sim_${util.uuidV4().substring(0, 8)}`,
+    suffix: '.py',
+  });
+  await fs.writeFile(tmpScript, APP_ACTIVATION_SCRIPT(pid), 'utf8');
+  try {
+    await exec('/usr/bin/python', [tmpScript]);
```
Comment (KazuCocoa) [inline lib/utils.js:320]: how about catch and print an error to install https://pypi.org/project/pyobjc/ ?
No module error will be raised if the machine has no PyObjC

Comment (mykola-mokhnach) [reply lib/utils.js:320]: Python with ObjC bindings is always included into MacOS by default

Comment (KazuCocoa) [reply lib/utils.js:320]: Okay
/usr/local/lib/python2.7/site-packages/ had AppKit, but system installed python3.x did not have. So I wondered I manually installed it in my past.

Comment (mykola-mokhnach) [reply lib/utils.js:320]: https://stackoverflow.com/questions/6819661/python-location-on-mac-osx

**Pull request merged**